src/js/components/DataTable/__tests__/DataTable-test.js

```diff
@@ -182,6 +182,42 @@ describe('DataTable', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('sort nested object', () => {
+    const { container, getByText } = render(
+      <Grommet>
+        <DataTable
+          columns={[
+            { property: 'a', header: 'A' },
+            {
+              property: 'b.value',
+              header: 'Value',
+              render: datum => datum.b && datum.b.value,
+            },
+          ]}
+          data={[
+            { a: 'zero', b: { value: 1 } },
+            { a: 'one', b: { value: 2 } },
+            { a: 'two', b: { value: 3 } },
+          ]}
+          sortable
```

Comment:
If you don't use sortable, and just use onSort, would that still work? can we add a test for that as well?

Reply:
Yeah, it will work, added a test for that.